modules/apps/change-tracking/change-tracking-web/npmscripts.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "build": {
+    "babel": false,
+    "bundler": false,
+    "exports": false,
+    "main": false,
+    "sass": false,
+    "tsc": false
+  }
+};
```

Comment:
Why is these two `npmscripts.config.js` files now committed? They look unrelated :thinking:

Reply:
Mmm it seems to be created by the toolset... I've checked for more garbage files and delete them.